charts/vault-secrets-operator/values.yaml

```diff
@@ -78,7 +78,7 @@ environmentVars: []
 # will watch all namespaces. If the value is empty and rbac.namespaced is set to
 # true, then the namespace of the release will be used.
 vault:
-  address: "http://vault:8200"
+  address: ""
```

Comment:
This might be breaking, open to different ways of doing this

Reply:
This should be ok, the next release also contains a breaking change for the CRD handling of the Helm chart.